docs/developer/process/agent-release.md

```diff
@@ -112,4 +117,4 @@ ddev agent changelog
 ddev agent integrations
 ```
 
-Create a pull request and wait for approval before merging.
+Update the [`AGENT_CHANGELOG`][agent-changelog] and [`AGENT_INTEGRATIONS`][agent-integrations] files. Create a pull request and wait for approval before merging.
```

Comment:
Yes, these commands just output stuff to the console so good point to instruct to actually update the files.

Nit copy suggestion to make this action item even more explicit:

```suggestion
Update the [`AGENT_CHANGELOG`][agent-changelog] and [`AGENT_INTEGRATIONS`][agent-integrations] files with the outputs of these commands. Create a pull request and wait for approval before merging.
```